vignettes/hubverse_ms_vignette.Rmd

```diff
@@ -0,0 +1,220 @@
+
+```{r, fig.cap = "Forecasts of hospital admissions in Texas due to influenza in the 2023-2024 respiratory virus season. Predictions from five submitted models and an ensemble are shown along with observed data. The observed data are shown as a line with dots for specific weekly observations. Each individual model is shown in orange and the mean ensemble is shown in black, with the 80% prediction intervals shown as shaded regions."}
+ensemble_forecast <- hubEnsembles::simple_ensemble(flusight_data)
+
+all_forecasts <- flusight_data |>
+  dplyr::bind_rows(ensemble_forecast)
+
+all_forecasts |>
+  hubVis::plot_step_ahead_model_output(target_data,
+                                       x_col_name = "target_end_date",
+                                       use_median_as_point = TRUE,
+                                       group = "reference_date",
+                                       interactive = FALSE,
+                                       fill_transparency = .2,
+                                       intervals = 0.90,
+                                       one_color = "darkgray",
+                                       ens_name = "hub-ensemble",
+                                       ens_color = "orange",
+                                       pal_color = NULL) +
+  ggplot2::theme(legend.position = "bottom") +
+  ggplot2::ylab("hospital admissions due to influenza")
+```
+
+\clearpage
+
+# Evaluate forecasts
+
+The package hubEvals can be used to evaluate probabilistic predictions against observed data. The hubEvals package creates an interface for hubverse-style model output and target data to the scoringutils package, which is an engine for computing a number of common scores for probabilistic predictions.[@bosse-scoringutils-2024] 
+
+The code below starts by making oracle output data, which is a dataset formatted like hubverse-style model output data but with a point prediction representing the eventual observation, as if observed by an oracle model. The FluSight hub stores target data in a time-series format but not in an oracle output format. The oracle output data format is convenient to use for scoring, as it can be joined with model output data and then the observation and the prediction reside in a single row of stored data. In this example, we compute and show results for the weighted interval score (WIS) with its associated decomposition into overprediction, underprediction, and dispersion penalties, the 50% and 90% prediction interval coverage rates, and the absolute error computed on the median prediction.[@bracher_evaluating_2021]
```

Comment:
Should we add a little bit more, by saying that you are using the `hubEvals::score_model_out()` function that scores model outputs with a single output_type against observed data, and more information is available in the help page of the function and the associated package?